NoPythonDemo/StyleKitSample/Info.plist

```diff
@@ -2,6 +2,8 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>StyleKit-DesignatedFolder</key>
```

Comment:
Let's maybe call it 'StylesheetLocation'? I also think it should be a directory + filename or maybe support both?